common-app-lib/src/cmr/common_app/services/search/acl_results_handler_helper.clj

```diff
@@ -1,13 +1,12 @@
-(ns cmr.search.services.acls.acl-results-handler-helper
+(ns cmr.common-app.services.search.acl-results-handler-helper
```

Comment:
You moved this class to common-app which is fine.  There is a different part of the code that calls (defmethod parse-elastic-item :granule, so you will have to update that name space too - I don't remember what it was.